metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache 2.0'
 description 'Installs and configures the collectd monitoring daemon.'
 long_description 'Installs and configures the collectd monitoring daemon.'
-version '2.2.2'
+version '2.2.3'
```

Comment:
for systems using the existing cookbook when converging and picking up this update, it will add the pidfile, trigger a service restart, and run via fork instead of foreground.